tests/integration/tests/namespace/test-resource-validation.spec.js

```diff
@@ -118,6 +128,11 @@ context('Test resource validation', () => {
 
     cy.get('[title="Profile"]').click();
 
+    cy.get('ui5-li:visible')
+      .contains('Preferences')
+      .invoke('show')
```

Comment:
in this file as well

Reply:
Form me without `.invoke('show')` it couldn't find `li`